images/zenko-zookeeper/kubernetes-entrypoint.sh

```diff
@@ -6,6 +6,15 @@ ZOO_HEAP_SIZE=${ZOO_HEAP_SIZE:-2G}
 ZOO_SERVER_PORT=2888
 ZOO_ELECTION_PORT=3888
 
+# Changes from kube to java friendly format 
```

Comment:
@Zempashi @ballot-scality Is there no way to do this at once in line 5, like `ZOO_HEAP_SIZE=${ZOO_HEAP_SIZE/i/}`, but then including the fallback?